app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms = @guest.rooms.includes(:hotel)
```

Comment:
We actually don't need to have line 4, since when we type `@guest.rooms` we should get back all rooms that are connected to this guest via the linking table. AR does this for us.